npm/private/npm_package_store.bzl

```diff
@@ -341,10 +348,6 @@ deps of npm_package_store must be in the same package.""" % (ctx.label.package,
     for target in ctx.attr.deps:
         npm_package_store_infos.append(target[NpmPackageStoreInfo])
 
-    for transitive_package_store_infos_depset in transitive_package_store_infos_depsets:
```

Comment:
Hmm, this was not needed i assume?

Reply:
It was replaced with `npm_package_store_infos.extend(jsinfo.npm_package_store_infos.to_list())` up above